embassy-stm32/src/can/bxcan.rs

```diff
@@ -92,17 +92,28 @@ foreach_peripheral!(
             const NUM_FILTER_BANKS: u8 = 14;
         }
     };
-    // Only correct when CAN2 also exists… Fix on yaml level?
-    // There are only 14 filter banks when CAN2 is not available.
+    // CAN1 and CAN2 is a combination of master and slave instance.
+    // CAN1 owns the filter bank and needs to be enabled in order
+    // for CAN2 to receive messages.
     (can, CAN1) => {
-        unsafe impl bxcan::FilterOwner for peripherals::CAN1 {
-            const NUM_FILTER_BANKS: u8 = 28;
+        cfg_if::cfg_if! {
+            if #[cfg(all(
+                any(stm32l4, stm32l72, stm32l73),
```

Comment:
there's probably some mistake here, STM32L7 is not a thing

Reply:
embarrassing mistake, indeed... fixed